src/helpers/identity.rs

```diff
@@ -0,0 +1,15 @@
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct HelperIdentity {
+    #[cfg(not(test))]
+    endpoint: hyper::Uri,
```

Comment:
I like enforcing constraints at compile time if it is at all possible. I am not sure that Uri is the right identity for a helper (this is still WIP) but assuming that we can start with it, I would prefer `Uri` rather than generic `String` that can really be anything

Reply:
well, i just meant there was no `new` fn or any actual constructor for the `HelperIdentity`

Reply:
true, once we move to config creation, we would know better how to initialize it in prod setting. For now test initialization is enough but it does not have to be that way.

Reply:
works for me